# LUTD-Base
LUTD-Base is an easy, interactive, and user-friendly shiny based web application that allows the other researcher to deeply explore the bulk RNA-Seq data, differential expression, functional enrichment analysis results, etc. from multiple LUTD related studies carried out by our group.

LUTD-Base web application is developed at [Functional Urology](http://www.urofun.ch/) lab at the [University of Bern](https://www.unibe.ch/index_eng.html/). The LUTD-Base can be accessed at https://functional-urology.unibe.ch/apps/LUTD-Base/.

# Application
LUTD-Base contains 5 main tabs, one for each LUTD subtype and a Functional Enrichment Exploration tab. Each tab consists of 3 different panels named Tables, Plots, GO Enrichment plots. Users can explore the raw read counts and edgeR results using the table panel. It allows users to explore the read counts and differential expression analysis results for the subset of genes using the plot panel. GO Enrichment plots provide static as well as interactive plots to explore functional enrichment analysis results. All the plots and tables are available with a download option.

# Citation
If LUTD-Base helps you in any way, please cite the following articles:

* Gheinani, Ali Hashemi et al. “Characterization of miRNA-regulated networks, hubs of signaling, and biomarkers in obstruction-induced bladder dysfunction.” JCI insight vol. 2,2 e89560. 26 Jan. 2017, doi: [10.1172/jci.insight.89560](https://insight.jci.org/articles/view/89560).
* Gheinani, Ali Hashemi et al. “Integrated mRNA-miRNA transcriptome analysis of bladder biopsies from patients with bladder pain syndrome identifies signaling alterations contributing to the disease pathogenesis.” BMC urology vol. 21,1 172. 7 Dec. 2021, doi: [10.1186/s12894-021-00934-0](https://bmcurol.biomedcentral.com/articles/10.1186/s12894-021-00934-0).

[I'm an inline-style link](https://www.google.com)
